projects/adrv9009/src/app/app_config.h

```diff
@@ -42,6 +42,7 @@
 
 //#define XILINX_PLATFORM
 //#define ALTERA_PLATFORM
+//#define ZU11EG
```

Comment:
i don't know if adding another define is the best approach. We are trying to limit this to a minimum, if we keep adding them we get back to where we started in the first place.

Reply:
I agree too many of these defines make the code hard to read and thought of what the alternative would be. One I can think of would be a separate project but this would lead to too much code duplication.